app.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "SideStore Offical",
-  "identifier": "com.SideStore.AltStore",
```

Comment:
Have you tested this that it doesn’t cause issues with installing and refreshing?

Reply:
I'll test this again

Reply:
This wouldn't cause issues with installing or refreshing as this is exclusively only for the update system and sidestore having a store app.

Reply:
https://github.com/SideStore/SideStore/actions/runs/3381406451 using this i confirmed it works